trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoSplitManager.java

```diff
@@ -30,11 +30,12 @@ public ConnectorSplitSource getSplits(
     GravitinoTransactionHandle gravitinoTransactionHandle =
         (GravitinoTransactionHandle) transaction;
 
+    // TODO(yuhui) add dynamic filter
     return internalSplitManager.getSplits(
         gravitinoTransactionHandle.getInternalTransactionHandle(),
         session,
         gravitinoTableHandle.getInternalTableHandle(),
-        dynamicFilter,
+        DynamicFilter.EMPTY,
```

Comment:
When will we use a non-null dynamicFilter?

Can you explain the real cause of the problem more clearly?

Reply:
In the current implementation of dynamicFilter, it includes ColumnHandle, and this columnhandle is currently GravitionColumnHandle. When it's passed to the following internal connector, such as PG, it will throw an error. The PG connector only accepts JDBCColumnHandle. Here, we pass DynamicFilter.EMPTY, which can temporarily solve this problem.